io.gitlab.azymohliad.WatchMate.yml

```
@@ -0,0 +1,37 @@
+id: io.gitlab.azymohliad.WatchMate
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: watchmate
+finish-args:
+  - --allow=bluetooth
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --system-talk-name=org.bluez
+  - --talk-name=org.mpris.MediaPlayer2.*
+  - --filesystem=xdg-download
+modules:
+  - name: watchmate
+    sources:
+      - type: git
+        url: https://gitlab.com/azymohliad/watchmate
+        commit: dc153113d6f31769e59dcbb238f5828fbd4f3a8b
+        tag: v0.2.0
+      - cargo-sources.json
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/rust-stable/bin
+      env:
+        CARGO_HOME: /run/build/watchmate/cargo
+    build-commands:
+      - cargo --offline fetch --manifest-path Cargo.toml --verbose
+      - cargo --offline build --release --verbose
+      - install -Dm 755 target/release/watchmate -t /app/bin
+      - install -Dm 644 assets/${FLATPAK_ID}.desktop -t /app/share/applications
+      - install -Dm 644 assets/${FLATPAK_ID}.metainfo.xml -t /app/share/metainfo
+      - install -Dm 644 assets/icons/${FLATPAK_ID}.svg -t /app/share/icons/hicolor/scalable/apps
+      - install -Dm 644 assets/icons/${FLATPAK_ID}-symbolic.svg -t /app/share/icons/hicolor/symbolic/app
```

I think you meant this:
```suggestion
      - install -Dm 644 assets/icons/${FLATPAK_ID}-symbolic.svg -t /app/share/icons/hicolor/scalable/apps
```

---

Oh, is that for sure? I can't find any documentation quickly, but I see [gtk-rust-template uses `symbolic/apps`](https://gitlab.gnome.org/World/Rust/gtk-rust-template/-/blob/master/data/icons/meson.build) for symbolic icon. I have it wrong anyway tho, at least need to change `app` to `apps`

---

It seems all other flatpak apps I have installed use `symbolic/apps` for symbolic icons, so I'll only fix `app` -> `apps` here for now. But let me know if it's actually incorrect, and I'll change to `scalable`.

```
$ find /var/lib/flatpak/app/**/apps -name "*-symbolic.svg" | grep symbolic | wc -l
240
$ find /var/lib/flatpak/app/**/apps -name "*-symbolic.svg" | grep scalable | wc -l
0
```